js/itbit.js

```diff
@@ -203,7 +203,8 @@ module.exports = class itbit extends Exchange {
             'cost': cost,
         };
         if (feeCost !== undefined && rebatesApplied !== undefined) {
-            if (feeCurrency === rebateCurrency) {
+            if (feeCurrency === rebateCurrency || rebateCurrency === null
```

Comment:
The check for `null` (change #3) is not needed, because `safeString` will return `undefined` a few lines above anyway.

So, there's no codepath where rebateCurrency can be `null`.